app/src/main/res/layout/activity_onboarding.xml

```diff
@@ -29,7 +29,6 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintHeight_max="700dp"
```

Comment:
The reason was to correctly align the toast with the card in the layouts. However it does look a bit nicer having the constraint in tablets. I have now moved the card to the onboarding activity layout to avoid the alignment issues and the constraint is back again.